packages/ui/components/ThemeSwitch.vue

```diff
@@ -53,6 +53,9 @@ export default {
   font-size: 20px;
   width: 20px;
   height: 20px;
+  &:hover {
```

Comment:
If we are making bigger the icon, I'd also make bigger the button where the icon is